tests/strings/002.phpt

```diff
@@ -3,7 +3,7 @@ Formatted print functions
 --FILE--
 <?php
 
-$fp = fopen("php://stdout", "w") or die("Arrggsgg!!");
+$fp = fopen("php://stdout", "w") or die("Arrrggg!!");
```

Comment:
I'm not sure about this. Is it just being cute? If so, is this cute enough?

Reply:
Just being cute you could change it to "Bug"